ansible/roles/eos/tasks/ceos_ensure_reachable.yml

```diff
@@ -0,0 +1,49 @@
+- block:
+  - name: wait for container's mgmt-ip reachable
+    wait_for:
+      port: 22
+      host: "{{ ansible_host }}"
+      search_regex: OpenSSH
+      delay: 0
+      timeout: 120
+    register: ip_result
+    ignore_errors: yes
+    delegate_to: "{{ VM_host[0] }}"
+
+  - name: restart cEOS container
+    when: ip_result.failed == true and 'Timeout when waiting' in ip_result.msg
+    become: yes
+    docker_container:
```

Comment:
This is to recreate the ceos container. Is `docker restart` sufficient?

Reply:
docker restart relies on the behavior of docker,
I think a native ansible module would be more rubost, avoiding mis-mount, setting loss. etc.

Reply:
Facing the same issue on ubuntu20.04 but not on ubuntu18.04. A restart helps, but it doesn't always fix the issue.

Reply:
My test shows that mgmt-ip unreachable issue happens both on 18.04 and 20.04, but the chance that issue happens on 18.04 is significantly less than 20.04.
For me a restart always make the mgmt-ip reachable, and is there any other issue you're facing?


> Facing the same issue on ubuntu20.04 but not on ubuntu18.04. A restart helps, but it doesn't always fix the issue.